### PR TITLE
[containerapp] `az containerapp job create`: Fix problem of parsing parameters `minExecutions` and `maxExecutions` from `--yaml`

### PR DESCRIPTION
**Pull request by anandanthony**
**Related command**
az containerapp job create

**Description**
[containerapp] `az containerapp job create`: Fix problem of parsing parameters `minExecutions` and `maxExecutions` from `--yaml`

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by Greedygre** [comment]
Recommend title:
[containerapp] `az containerapp job create`: Fix problem of parsing parameters `minExecutions` and `maxExecutions` from `--yaml`
#

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerappjob_executions.py [hunk 1]
```diff
@@ -556,8 +556,8 @@ def test_containerappjob_eventtriggered_create_with_yaml(self, resource_group, l
                             replicaCompletionCount: 2
```
Comment (Greedygre) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerappjob_executions.py:556]: Please rerun the test.

**Review by wangzelin007 — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/recordings/test_containerappjob_eventtriggered_create_with_yaml.yaml [hunk 2]
```diff
@@ -962,8 +963,8 @@ interactions:
 - request:
     body: '{"location": "eastus", "tags": null, "properties": {"daprAIInstrumentationKey":
       null, "vnetConfiguration": null, "appLogsConfiguration": {"destination": "log-analytics",
-      "logAnalyticsConfiguration": {"customerId": "5eed0cab-14d0-481a-8cd0-564db4196f2f",
-      "sharedKey":"abc123"}},
+      "logAnalyticsConfiguration": {"customerId": "7b383acd-1e53-4015-aadd-7bdc86c3fce0",
+      "sharedKey": "fZYNkrP9e30J6Mm0KqOyr8iqoEZUB3eRvvhCPqdEtzevJDij03zL4widoOozmq0gM5gSpY5dpWyCRfQnm3ugxA=="}},
```
Comment (wangzelin007) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/recordings/test_containerappjob_eventtriggered_create_with_yaml.yaml:967]: Please replace all shared keys with fake keys.

**Comment by wangzelin007** [comment]
Please replace all shared keys with fake keys to pass `Credential Scanner`.

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/recordings/test_containerappjob_eventtriggered_create_with_yaml.yaml (hunk 2 above)
Comment (Greedygre) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/recordings/test_containerappjob_eventtriggered_create_with_yaml.yaml:967]:
```suggestion
      "sharedKey": "abc123"}},
```

**Review by Greedygre — APPROVED** [review]
Please fix the credential scanner, otherwise LGTM.

**Review by wangzelin007 — APPROVED** [review]

**Pull request merged**